.travis.yml

```diff
@@ -0,0 +1,19 @@
+os:
+  - linux
+  - osx
+language: generic
+sudo: required
+dist: trusty
+osx_image: xcode7.3
+install:
+  - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/02090c7ede5a637b76e6df1710e83cd0bbe7dcdf/swiftenv-install.sh)"
+script:
+  - sudo cp vapor /usr/local/bin/vapor # install
+  - ./vapor self-update # self-update, will overwrite `./vapor`
```

Comment:
Can we add to this comment that calling `vapor self-update` would update the bin before testing and it would be problematic.

(Ideally more concisely than that!)